.github/workflows/cfn-publish-resource.yaml

```diff
@@ -0,0 +1,87 @@
+name: Atlas CFN Resource Publish
+on:
+  workflow_dispatch:
+    inputs:
+      regions:
+        description: "comma seperated AWS regions.Default is all regions."
+        default: "us-east-1,us-west-1,us-west-2,eu-west-1,eu-central-1,ap-southeast-1,ap-southeast-2,ap-northeast-1,ap-northeast-2,sa-east-1,ca-central-1,eu-west-2,eu-west-3,eu-north-1,ap-south-1,ap-east-1,me-south-1,af-south-1,us-east-2,us-west-1,us-west-2,eu-west-1,eu-central-1,ap-southeast-1,ap-southeast-2,ap-northeast-1,ap-northeast-2,sa-east-1,ca-central-1,eu-west-2,eu-west-3,eu-north-1,ap-south-1,ap-east-1,me-south-1,af-south-1,us-east-2"
```

Comment:
i think you are missing the new regions ap-south-2, ap-southeast-4, eu-central-2, eu-south-2, me-central-1

Reply:
@Adelmar92 : Added above regions